oldprovider/value_cannot_be_changed_modifier.go

```diff
@@ -4,23 +4,20 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema/planmodifier"
 
 	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
 	"github.com/hashicorp/terraform-plugin-log/tflog"
 )
 
-type valueCannotBeChangedModifier struct {
-}
-
-func ValueCannotBeChangedModifier() tfsdk.AttributePlanModifier {
-	return valueCannotBeChangedModifier{}
-}
-func (m valueCannotBeChangedModifier) Description(ctx context.Context) string {
-	return "The value cannot be changed after the resource was created."
-}
-
-func (m valueCannotBeChangedModifier) MarkdownDescription(ctx context.Context) string {
-	return m.Description(ctx)
+func StringValueCannotBeChanged() planmodifier.String {
+	return RequiresReplaceIf(
+		func(_ context.Context, _ planmodifier.StringRequest, resp *RequiresReplaceIfFuncResponse) {
+			resp.RequiresReplace = true
+		},
+		"If the value of this attribute changes, Terraform will destroy and recreate the resource.",
```

Comment:
This will spawn warning or fail apply?

Reply:
Removed that file - it's not needed anymore